metrics/web/lib/auth/domain/usecases/parameters/user_id_param.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:meta/meta.dart';
+
+/// A class that represents the user identifier parameter.
+@immutable
+class UserIdParam {
+  /// An user identifier.
+  final String id;
+
+  /// Creates a new instance of the [UserIdParam].
+  ///
+  /// The [id] should not be `null`.
```

Comment:
```suggestion
  /// The [id] must not be `null`.
```
To make it match similar documentation in other classes.